### ISSUE TITLE
postgresql: ERROR: relation "requests" already exists

### ISSUE THREAD

**Issue by holzman** [ssl-hep/ServiceX#624]
On a fresh helm install (provided I comment out the `db.init_app(app)` in `app.py` as mentioned in #623):

```
postgresql 14:33:15.10 INFO  ==> ** Starting PostgreSQL **                                                                                                               
2023-08-07 14:33:15.114 GMT [1] LOG:  pgaudit extension initialized                                                                                                      
2023-08-07 14:33:15.130 GMT [1] LOG:  starting PostgreSQL 14.4 on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit                         
2023-08-07 14:33:15.130 GMT [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432                                                                                     
2023-08-07 14:33:15.130 GMT [1] LOG:  listening on IPv6 address "::", port 5432                                                                                          
2023-08-07 14:33:15.130 GMT [1] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"                                                                                      
2023-08-07 14:33:15.133 GMT [131] LOG:  database system was shut down at 2023-08-07 14:33:14 GMT                                                                         
2023-08-07 14:33:15.137 GMT [1] LOG:  database system is ready to accept connections                                                                                     
2023-08-07 14:34:10.384 GMT [195] ERROR:  relation "requests" already exists                                                                                             
2023-08-07 14:34:10.384 GMT [195] STATEMENT:                                                                                                                             
    CREATE TABLE requests (                                                                                                                                              
        id SERIAL NOT NULL,                                                                                                                                              
        submit_time TIMESTAMP WITHOUT TIME ZONE NOT NULL,                                                                                                                
        did VARCHAR(512) NOT NULL,                                                                                                                                       
        columns VARCHAR(1024),                                                                                                                                           
        selection VARCHAR(10485760),                                                                                                                                     
        tree_name VARCHAR(512),                                                                                                                                          
        request_id VARCHAR(48) NOT NULL,                                                                                                                                 
        image VARCHAR(128),                                                                                                                                              
        chunk_size INTEGER,                                                                                                                                              
        workers INTEGER,                                                                                                                                                 
        result_destination VARCHAR(32) NOT NULL,                                                                                                                         
        result_format VARCHAR(32) NOT NULL,                                                                                                                              
        kafka_broker VARCHAR(128),                                                                                                                                       
        workflow_name VARCHAR(40) NOT NULL,                                                                                                                              
        files INTEGER,                                                                                                                                                   
        files_skipped INTEGER,                                                                                                                                           
        total_events BIGINT,                                                                                                                                             
        total_bytes BIGINT,                                                                                                                                              
        did_lookup_time INTEGER,                                                                                                                                         
        generated_code_cm VARCHAR(128),                                                                                                                                  
        PRIMARY KEY (id),                                                                                                                                                
        UNIQUE (request_id)                                                                                                                                              
    )                                                                                  
    ```

**Comment by holzman** [comment]
FWIW, this looks like the first migration.  Is this some conflict where the migration expects an empty db, but the db creation now creates the tables?

**Comment by BenGalewsky** [comment]
Fixed via #630
